development/tools/pkg/orphanremover/remover.go

```diff
@@ -0,0 +1,313 @@
+package orphanremover
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	compute "google.golang.org/api/compute/v1"
+)
+
+const sleepFactor = 2
+
+//go:generate mockery -name=ComputeAPI -output=automock -outpkg=automock -case=underscore
+
+//ComputeAPI interface logic for Google cloud API
+type ComputeAPI interface {
```

Comment:
IMO all methods here should return an error.
In the ideal world, errors are propagated to the `main` function and only there are logged.